README.md

```diff
@@ -40,25 +38,37 @@ Auth SDK enables the standard [Ethereum provider](https://eips.ethereum.org/EIPS
 
 - Passwordless authentication
 
-Auth SDK usage is easily customizable. Developers can choose to use the built-in plug and play login UI to onboard users or use a custom UI. Also, instead of the built-in Arcana wallet UI, developers can integrate with the SDK and use a custom wallet UI.
+## Supported Blockchain Networks
+
+The Arcana wallet is [pre-configured with a subset of supported chains](https://docs.arcana.network/state-of-the-arcana-auth#supported-blockchains). Others can be added programmatically or via the wallet UI.
```

Comment:
The dashboard?

Reply:
hmm, yes but by the devs. Ok let me rephrase it.